frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -139,6 +139,7 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
                 isRequired
                 type="text"
                 id="pvc-name"
+                data-cy="pvc-name"
```

Comment:
Why you need to add `data-cy` if `id` is already present? You can use `id` to select the pvc-name

Reply:
I tried to follow best practices mentioned here: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements.